.goreleaser.yaml

```diff
@@ -0,0 +1,51 @@
+---
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # Keep mod files clean
+    - go mod tidy -v
+
+builds:
+  - main: main.go
+    binary: bin/diff
+    env:
+      - CGO_ENABLED=0
+    flags: -trimpath
+    ldflags: -s -w -X {{ .ModulePath }}/cmd.Version={{ .Version }}
+    targets:
+      # helm supported targets
+      # https://github.com/helm/helm/blob/main/Makefile#L4C1-L4C1
+      - darwin_amd64
+      - darwin_arm64
+      - linux_386
+      - linux_amd64
+      - linux_arm
+      - linux_arm64
+      - linux_ppc64le
+      - linux_s390x
+      - windows_amd64
+      # additional targets
+      - freebsd_amd64
+
+archives:
+  - format: tgz
+    name_template: '{{ .ProjectName }}-{{ tolower .Os }}-{{ .Arch }}'
+    wrap_in_directory: diff
+    files:
+      - README.md
+      - LICENSE
+      - plugin.yaml
+
+checksum:
+  name_template: 'checksums.txt'
+
+snapshot:
+  name_template: '{{ incpatch .Version }}-{{ .ShortCommit }}'
+
+changelog:
```

Comment:
I missed this in my PR so let me check later!

Reply:
Doesn't required for releases, but I like this snapshot versioning